src/main/scala/chisel3/iotesters/ChiselMain.scala

```diff
@@ -146,7 +146,7 @@ object chiselMain {
       case "firrtl" =>
         val file = new java.io.File(context.targetDir, s"${dut.name}.ir")
         val ir = io.Source.fromFile(file).getLines mkString "\n"
-        new FirrtlTerpBackend(dut, ir, context.testerSeed)
+        new FirrtlTerpBackend(dut, ir)
```

Comment:
We probably want to give a specific seed for debugging. Can we give a seed as an option?